.markdownlint.json

```diff
@@ -7,11 +7,7 @@
   "no-trailing-punctuation": false,
   "no-trailing-spaces": false,
   "no-multiple-blanks": false,
-  "MD005": false,
-  "MD007": false,
-  "MD010": false,
-  "MD011": false,
-  "MD018": false,
+  "MD007": { "indent": 4 },
```

Comment:
The indentation level needed to be set to 4. The value of 2 was not enough to guarantee that nested lists appear correctly.